math-native/src/main/java/org/ballerinalang/stdlib/math/nativeimpl/Constant.java

```diff
@@ -35,9 +35,15 @@
 
     public static final String OVERFLOW_ERROR_MSG = "Overflow occurred";
 
-    private static final String PACKAGE_NAME = "math";
-
+    /**
+     * Math standard library version.
+     * @deprecated Use MathUtils.getMathPackage().getVersion().
+     */
     public static final String MATH_PACKAGE_VERSION =  "1.0.1";
```

Comment:
We can remove this completely right? Are we using this anywhere else other than the math module?

Reply:
Removed in 3e935cb